packages/block-library/src/image/edit.native.js

```diff
@@ -244,6 +268,15 @@ class ImageEdit extends React.Component {
 					autoCorrect={ false }
 					keyboardType="url"
 				/>
+				<BottomSheet.Cell
+					icon={ 'editor-expand' }
+					label={ __( 'Size' ) }
+					labelStyle={ styles.sizeInspectorControlsCell }
```

Comment:
This style seems to have been removed [here](https://github.com/WordPress/gutenberg/pull/17245/commits/ef79f0852f250b6e04d3ba7d95deb65d4d47fc05).
Is this still needed?

Reply:
Thanks for catching that, removed here: ad81e1596f6b625a5df7641aa5d48591a278768d